testing/cypress/e2e/smoke-6-deleterequest.cy.ts

```diff
@@ -0,0 +1,20 @@
+// This spec is used to navigate to the request detail page for a specific request.
+// It is used to verify that the request detail page is not broken and to verify that basic navigation is possible.
+
+import Request from "../appActions/Request";
+const requestID = '00008882';
```

Comment:
@rstens are you planning to remove this hardcoded one? because once the request is deleted, you would not be able to access it in next test run

Reply:
Yes, this is merely temporary